Tests/Client/Unit Tests/Client+DevicesTests.swift

```diff
@@ -53,6 +57,36 @@ extension Client_DevicesTests {
             body: nil
         )
     }
+
+    func test_getDevice_handlesSuccess() throws {
+        // Setup
+        let deviceId = "device_id_\(UUID().uuidString)"
+        let timestamp = Date(timeIntervalSince1970: 123456789)
+
+        MockNetworkURLProtocol.mockResponse(endpoint: .devices(testUser), responseBody:
+            ["devices": [ ["id": deviceId, "created_at": ISO8601DateFormatter().string(from: timestamp)] ]]
+        )
+
+        // Action
+        let result = try await { self.client.devices($0) }
+
+        // Assert
+        AssertResultSuccess(result, [Device(deviceId, created: timestamp)])
+                
+        AssertAsync.willBeEqual(self.client.user.devices, [Device(deviceId, created: timestamp)])
```

Comment:
By the time line
`let result = try await { self.client.devices($0) }`
completes, `client.user.devices` would already be set to the correct value, so this doesn't need to be `async`

Reply:
Hmm, that's probably true. Once the completion is called, the values have to be set. 👍

Reply:
Fixed. I left it there for the inversed expectation to guard ourselves against potential off-main-queue errors.